core/src/test-integration/java/io/neow3j/protocol/Neow3jExpressIntegrationTest.java

```diff
@@ -40,44 +41,41 @@
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class Neow3jExpressIntegrationTest {
 
+    static final String NEF_FILE_SOURCE = "/OracleTestContract.nef";
+    static final String NEF_FILE_DESTINATION = "/app/OracleTestContract.nef";
+    static final String MANIFEST_FILE_SOURCE = "/OracleTestContract.manifest.json";
+    static final String MANIFEST_FILE_DESTINATION = "/app/OracleTestContract.manifest.json";
+    static final String INVOKE_FILE_SOURCE = "/oracle-request-invoke.json";
+    static final String INVOKE_FILE_DESTINATION = "/app/oracle-request-invoke.json";
+
+    static final Hash160 oracleTestContractHash = new Hash160(
+            "8693a8d7d9c5633f9b27cb4366f98b4171dc8838");
+
+    static Hash256 oracleRequestTx;
+
     protected static Neow3jExpress neow3jExpress;
 
     @ClassRule
-    public static NeoTestContainer neoTestContainer = neoExpressTestContainer(1);
+    public static NeoExpressTestContainer neoTestContainer = new NeoExpressTestContainer(1,
+            NEF_FILE_SOURCE, NEF_FILE_DESTINATION,
+            MANIFEST_FILE_SOURCE, MANIFEST_FILE_DESTINATION,
+            INVOKE_FILE_SOURCE, INVOKE_FILE_DESTINATION
+    );
```

Comment:
I couldn't find this new class `NeoExpressTestContainer`. Please push it as soon as it's ready. :+1: